task_4/andrii.lukianchykov/task_4/src/main/java/com/bookshop/bookdatabase/BookDataBase.java

```diff
@@ -0,0 +1,159 @@
+package com.bookshop.bookdatabase;
+
+import static java.lang.System.out;
+
+import com.bookshop.book.Book;
+import com.bookshop.helpTests.IncorrectSymbol;
+import com.bookshop.helpTests.ListIsEmptyException;
+import com.bookshop.helpTests.ListIsNotEmptyException;
+import com.bookshop.helpTests.ListIsNullException;
+import com.bookshop.response.Response;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class BookDataBase {
+
+    private List<Book> listOfBooks = new ArrayList<>();
+    private Book book = new Book();
+    private Response response = new Response();
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
```

Comment:
You can use List's **toString()** method. No need to wrap it up in a StringBuilder.